VoteCounter/src/Player.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace VoteCounter
+{
+    class Player
+    {
+        public int numberOfVotes { get; set; }                                            //number of votes player has
+      //  { get { return this.numberOfVotes; } set { this.numberOfVotes = value; } }                                               
+        public List<string> nicknameList { get; set; }                                                    //all names player goes by. 
+            // { get { return this.nicknameList; } set { this.nicknameList = value; } }                                    
+        public string mainName { get; set; }                                                               //forum name. 
+              //  { get {return this.mainName; } set {this.mainName = value; } }                                                
+        public DateTime lastPost { get; set; }                                                             //the last day&time this player posted.
+            //   { get {return this.lastPost; } set {this.lastPost = value; } }                                               
+
+        public  Player()
+        {
+            numberOfVotes = 1;
+            mainName = ("Foobar");
+            lastPost = new DateTime();
+        }
+
+        public Player(Player p)
+            {
+            this.numberOfVotes = p.numberOfVotes;
+            this.mainName = p.mainName;
+            lastPost = p.lastPost;
+            nicknameList = p.nicknameList;
+        }
+
+        public Player(string mainName, int numberOfVotes, List<string> nicknames)
```

Comment:
Ah. I was wondering why it was throwing errors when I tried an initializer list. I'll look at that more closely soonish